src/Scene.js

```diff
@@ -15,6 +15,64 @@ type SceneProps = {
 };
 
 class Scene extends React.Component<SceneProps, {}> {
+
+    drawGrid(sceneWidth: number, sceneHeight: number, minX: number, minY: number) {
+        const grid = [];
+        const rowLabelOffset = sceneWidth * 0.025;
+        const columnLabelOffset = sceneHeight * 0.025;
+        let yOffset = minY;
+        if (this.props.numRows === 0 || this.props.numColumns === 0) {
+            return grid;
+        }
```

Comment:
We can move the `if (this.props.numRows === 0 || this.props.numColumns === 0)` test up earlier in the method, underneath

    const grid = [];